tests/test_indentscope.lua

```diff
@@ -509,6 +509,21 @@ T['draw()']['does not round time of every animation step'] = function()
   child.expect_screenshot()
 end
 
+T['draw()']['shows symbols on wrapped lines'] = function()
+  -- Add a long line of text. Set 'breakindent' and 'breakindentopt' to ensure
+  -- the wrapped line has the same indent as the current line
+  set_cursor(6, 1)
+  child.cmd('normal! 10Aa')
```

Comment:
Maybe `set_lines()` with some smaller set of lines together with explicit `child.set_size()` makes the test clearer?

Reply:
b55794d6f958faf4d101bfb0cefb3943cc794311